src/main/java/com/example/waggle/global/util/ParseUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.example.waggle.global.util;
+
+import com.example.waggle.domain.conversation.persistence.entity.Conversation;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ParseUtil {
+
+    private static final String mentionRegex = "@\\[([^]]+)]\\(([^)]+)\\)";
+
+    public static List<String> parsingUserUrl(Conversation conversation) {
+        Pattern pattern = Pattern.compile(mentionRegex);
+        Matcher matcher = pattern.matcher(conversation.getContent());
+        List<String> userUrlList = new ArrayList<>();
+        while (matcher.find()) {
+            userUrlList.add(matcher.group(2));
+        }
+        return userUrlList;
```

Comment:
```suggestion
     private static final String MENTION_REGEX = "@\\[([^]]+)]\\(([^)]+)\\)";
    private static final Pattern PATTERN = Pattern.compile(MENTION_REGEX);

    public static List<String> extractUserUrlsFromConversation(Conversation conversation) {
        if (conversation == null || conversation.getContent() == null) {
            log.warn("NPE");
            return new ArrayList<>();
        }
        Matcher matcher = PATTERN.matcher(conversation.getContent());
        List<String> userUrlList = new ArrayList<>();
        while (matcher.find()) {
            userUrlList.add(matcher.group(2));
        }
        return userUrlList;
```

이런식으로 정규식을 클래스 로딩할 때 한 번만 컴파일해서 static 필드로 저장해도 괜찮을 것 같습니다! (상수는 대문자로 부탁드려요 !!)

Reply:
좋네요 코드가 더 보기 좋은것 같습니다 수정해서 올릴게요